swift-extensions/TrikotHttpResponse.swift

```diff
@@ -22,8 +23,8 @@ public class TrikotHttpResponse: NSObject, HttpResponse {
             }
             statusCode = Int32(response.statusCode)
         }
-        if let data = data {
-            bodyString = String(data: data, encoding: .utf8)
+        if let data = data, data.count > 0 {
```

Comment:
On a second tought, I think we should have a ByteArray(0) instead of nil in our bodyArray.

Else if 👍